src/bb2.py

```diff
@@ -2,13 +2,90 @@
 Blue Button 2.0 SDK Class
 
 """
+import json
+import os
+import pathlib
+import yaml
+
+ROOT_DIR = os.path.abspath(os.curdir) + "/"
+DEFAULT_CONFIG_FILE_LOCATION = ROOT_DIR + "./.bluebutton-config.json"
+
+ENVIRONMENT_URLS = {
+    "SANDBOX": "https://sandbox.bluebutton.cms.gov",
+    "PRODUCTION": "https://api.bluebutton.cms.gov",
+}
 
 
 class Bb2:
```

Comment:
Think we could also change this to `BlueButton`? Just to give consistency across all the projects

Reply:
@nbragdon Keep forgetting to change this. Just changed it :-)